contrib/pax_storage/doc/README.md

```diff
@@ -21,7 +21,26 @@ PAX has the following features:
 
 ### Build PAX
 
-PAX will be built with `--enable-pax` when users building the Cloudberry.
+PAX will be built with `--enable-pax` when users building the Cloudberry. Dependency requirements are as follows:
+
+- **C/C++ Compiler**: GCC/GCC-C++ 11 or later
+- **CMake**: 3.11 or later
+- **Protobuf**: 3.5.0 or later
+- **ZSTD (libzstd)**: 1.4.0 or later
+- **glib-2.0** (option, only for vectorization)
```

Comment:
vectorization is off in cloudberry and we have no way to turn it on.

Reply:
Good catch. So we can remove this line now, right? 
If we contribute this feature back to Cloudberry, we can add this later.